lucien-ios/AddNewComicBook/AddComicViewController.swift

```diff
@@ -99,31 +104,27 @@ final class AddComicViewController: UIViewController {
     }
 
     private func configureViewController() {
-        // Comic Title
-        configureTextFieldBorder(textField: comicTitleTextField)
-        comicTitleTextField.attributedPlaceholder = createPlaceHolderAttributedString(placeholder: "Comic Title")
-        comicTitleTextField.addTarget(self, action: #selector(AddComicViewController.comicTitleEditingChanged), for: .editingChanged)
-
-        // Story Title
+        // Cover Photo Button
+        // The attributed title for the coverPhotoButton is set on the storyboard, but if an attributed title is created for the highlighted state
+        // the text in the highlighted state will shift some reason. In order to avoid this, an attributed title is created for both
+        // the highlighted and normal state. The textAlignment and lineBreakMode are also set.
+        let coverPhotoButtonTitleNormal = NSAttributedString(string: "Take Cover Photo", attributes: [NSAttributedStringKey.font: LucienTheme.Fonts.muliBold(size: 16.0) ?? UIFont(), NSAttributedStringKey.foregroundColor: UIColor.white])
```

Comment:
Pull all the cover photo button setup stuff out into a function called `setupCoverPhotoButton` and call that here.

Reply:
Also looks like the image and text are a little off center?

Reply:
I'll take a look into that centering issue right now.